infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -184,5 +184,11 @@
     "id": 32,
     "reason": "Unable to deserialize Filter definition.",
     "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ServiceLimitException",
+    "id": 31,
```

Comment:
Nitpick, should be above the 32 to keep order.

Reply:
done.. fix the ordering.